quipucords/scanner/network/utils.py

```diff
@@ -92,23 +92,33 @@ def write_inventory(inventory):
     return write_to_yaml(inventory)
 
 
-def create_ansible_objects(inventory_file, extra_vars, forks=50):
+def create_ansible_objects(inventory_file, extra_vars, use_paramiko=False,
+                           forks=50):
     """Create the default ansible objects needed to run a playbook.
 
     :param inventory_file: The path to the inventory file
     :param extra_vars: The dictionary containing
            the collection status of the optional products.
+    :param use_paramiko: use paramiko instead of ssh for connection
     :param forks: number of forks to run with, default of 50
     :returns: tuple of (options, variable_manager, loader, inventory)
     """
     named_options = namedtuple('Options', ['connection', 'module_path',
                                            'forks', 'become', 'become_method',
                                            'become_user', 'check'])
-    options = named_options(connection='ssh',
-                            module_path=C.DEFAULT_MODULE_PATH,
-                            forks=forks, become=False,
-                            become_method='sudo', become_user='root',
-                            check=False)
+    if use_paramiko:
+        # Change connection method from default ssh to paramiko
+        options = named_options(connection='paramiko',
```

Comment:
instead of rebuilding the options based on whether or not we use paramiko, can we just have a variable for connection that defaults to ssh, but if `use_paramiko` is True, we change it to `paramiko`?